### ISSUE TITLE
Isaac Cameron Boss Art

### ISSUE THREAD

**Issue by goofanader** [goofanader/minna-no-fighting-game#3]
## Checklist
- [x] Front Facing Sprite
- [ ] Idle Animation
- [ ] Point to the Right Sprite
- [x] Attack 1 Animation
### Optional Animations
- [ ] Walking Animation
- [ ] Force Blast Sprite (hands and arms out to the sides of the screen, as if force pushing people away)
- [ ] Attack 2 Animation
## References
- **Boss Name:** The Cat King
- **References:** [Hair Reference](https://drive.google.com/file/d/0B0GuMMchYnYOODZLRU13dUZGcXM/view?usp=sharing), [Clothing Reference](https://drive.google.com/file/d/0B0GuMMchYnYOaG12WGdGOTg3c043VVdZZXBLTVlzWW9WTEZv/view?usp=sharing)
- **Opening Monologue:** I regret nothing!
## Additional Items/Changes to the Reference
- **Hair:** Black curly hair, just not too much to encroach on Elliott's afro turf (ha). It can blow back and appear longer during my special. I did used to have long hair.
- **Face:** I typically have a beard; hopefully it won't be too out of place with the plugsuit.
## Colors
- **Primary:** white
- **Secondary:** dark blue
- **Tertiary:** see suit
### Regular Attack: TK and Stuff

Objects float up around me and then get thrown at people. Most of them are probably construction related. I guess sort of like this? I've also included a bunch of possible objects.
#### References
- [Accelerator from A Certain Scientific Railgun](https://youtu.be/tLU3Lw29biU?t=1m29s)
- [Steel Construction Manual](http://www.arch.illinois.edu/sites/default/files/styles/portfolio_image/public/faculty-portfolio-gallery/2_1.jpg?itok=3fHcf85t http://www.rcpblock.com/images/products/block/precision-product-natural.jpg)
- [International Building Code Book](http://ppi2pass.com/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/i/b/ibc12_magento.png)
- [Cordless Drill](https://upload.wikimedia.org/wikipedia/commons/1/1e/CordlessDrill.jpg)
- okay, I'll stop...

**Phyllis Note:** Isaac is an architectural engineer major. Hence all the architecture stuff...
### Special Attack: CATtack on Minna

I stand up tall and face away from the player, holding up a can of Friskies, then yell "Alright, dinner break everyone! (nyan)". A torrent of cats run across the screen. Since there's no jumping, the player can probably just knock each one out of the way with a single hit.
## Last Remarks

Let me know if anything is too hard, or is just plain stupid.

**Comment by eyfey** [comment]
![isaac](https://cloud.githubusercontent.com/assets/11142143/11553729/622dc980-9946-11e5-85dc-07711ff1f50a.png)
pretty sure I'm going to touch this one up a bit later- but here it is for now

**Comment by eyfey** [comment]
![isaacsscjorthair](https://cloud.githubusercontent.com/assets/11142143/12159448/29d93ada-b497-11e5-858d-cc990f85d593.png)
shorter hair! Should I go shorter?

**Comment by goofanader** [comment]
yeah, go shorter. I don't think there should be that much below his chin imo

**Comment by eyfey** [comment]
![isaacsscjorthair](https://cloud.githubusercontent.com/assets/11142143/12159602/830f23de-b498-11e5-8e24-33192d298089.png)
 How's that? Or shorter?

**Comment by goofanader** [comment]
yeah that's good
I'd go with that!

**Comment by goofanader** [comment]
![isaac](https://cloud.githubusercontent.com/assets/799112/12852527/3a4d7886-cbe4-11e5-8796-7edd12a51080.png)

Isaac's temp spritesheet. It's just a sketch, but the items are done.
...You can't see his body 'cause it's white....

**Comment by eyfey** [comment]
![isac](https://cloud.githubusercontent.com/assets/11142143/12869031/db9c716c-ccc8-11e5-8923-49a747b3e2e2.png)

boop bop
